elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -179,6 +195,42 @@ default boolean enableJSONAPI() {
     default boolean enableGraphQL() {
         return true;
     }
+    
+    /**
+     * Enable the support for Async querying feature. If false, the async feature will be disabled.
+     *
+     * @return Default: False
+     */
+    default boolean enableAsync() {
+        return false;
+    }
+
+    /**
+     * Thread Size for Async queries to run in parallel.
+     *
+     * @return Default: 5
+     */
+    default Integer getAsyncThreadSize() {
+        return 5;
+    }
+
+    /**
+     * Maximum Query Run time for Async Queries to mark as TIMEDOUT.
+     *
+     * @return Default: 60
+     */
+    default Integer getMaxRunTime() {
+        return 60;
+    }
+
+    /**
+     * Number of hosts running Elide Service.
+     *
+     * @return Default: 1
+     */
+    default Integer getNumberOfHosts() {
```

Comment:
I don't think we need this.

Reply:
We are using this param to randomize the start-up time for the cleanup executor. If we have more than 1 host running and then if all of them have the cleanup service executing at the same time they could possibly try to update/delete the record at the same time and have lock error etc.  so this is how we are using it here. https://github.com/yahoo/elide/blob/c88d08a94c994f16ee51c6a51c39694411fb8c59/elide-async/src/main/java/com/yahoo/elide/async/service/AsyncExecutorService.java#L59-L63

Reply:
done